src/components/organisms/product-variant-tree/index.tsx

```diff
@@ -62,9 +64,9 @@ const ProductVariantLeaf = ({ sku, title, prices = [] }: LeafProps) => {
       </div>
       <div className="flex items-center text-grey-50 flex-1 justify-end">
         <div className="text-grey-50 mr-xsmall">
-          {prices.length ? (
+          {filteredPrices.length ? (
             <span>{`${prices.length} price${
```

Comment:
comment: Found the culprit 😄 
```suggestion
            <span>{`${filteredPrices.length} price${
```

Reply:
haha shit nice catch 😆